oe_corporate_blocks.module

```diff
@@ -6,3 +6,66 @@
  */
 
 declare(strict_types = 1);
+
+/**
+ * Implements hook_theme().
+ */
+function oe_corporate_blocks_theme() {
+  return [
+    'corporate_blocks_footer' => [
```

Comment:
On components we should prefix all our theme entries with `oe_`, I suggest to change this into `oe_corporate_footer`.

Reply:
renamed